R/PackageInheritanceReporter.R

```diff
@@ -0,0 +1,212 @@
+#' @title Package Class Inheritance Reporter Class
+#' @name InheritanceReporter
+#' @family PackageReporters
+#' @description This reporter takes a package and traces the class inheritance
+#'   structure. Currently the following object-oriented systems are supported:
```

Comment:
props to you for taking on all three! If I had picked up this issue, I totally would have just started with R6 and said "the others are coming later"